lib/shortcode.php

```diff
@@ -185,38 +182,40 @@ function anvato_shortcode($attr) {
 		$json['html5'] = true;
```

Comment:
This code was originally present. We have not added it.
This is the link to Master https://github.com/cbslocal/wp-anvato-plugin/blob/master/lib/shortcode.php#L174 where html5 parameter is set.
On line 186 I have commented it out, since it breaks out Live Streams, but its not optimal.

Please advise.